httplib2_transport/README.rst

```diff
@@ -0,0 +1,32 @@
+Httplib2 Transport for Google Auth
+==================================
+
+|pypi|
+
+This library provides an `httplib2`_ transport for `google-auth`_.
+
+.. note:: Httplib2 has lots of problems. Using it is highly discouraged. This
+    library is intended to help existing users of oauth2client migrate to
+    google-auth.
```

Comment:
Oh snap! (also, suggest code-font for relevant code-y nouns)

Also, suggest making at least one such "problem" concrete here: "for example, it uses a connection cache that is not threadsafe"